Resources/Prototypes/_RMC14/Entities/rmc_planets.yml

```
@@ -129,7 +129,7 @@
 
 - type: entity
   id: RMCPlanetShiva
-  name: Shivas Snowball
+  name: LV-631 Shivas Snowball
```

```suggestion
  name: LV-343 Shivas Snowball
```
This is the planet name from CM13

---

its called lv-631 in our metadata for some reason, I can't find either LV-631 or LV-343 in the CM code however